dspace-api/src/main/java/org/dspace/core/Context.java

```diff
@@ -944,4 +948,8 @@ public String getAuthenticationMethod() {
     public void setAuthenticationMethod(final String authenticationMethod) {
         this.authenticationMethod = authenticationMethod;
     }
+
+    public boolean isSwitchedContextUser() {
+        return switchedContextUser;
```

Comment:
This method needs a description to explain that it has to do with whether `switchContextUser()` has been called on this Context object.  Without that description, it doesn't really make sense.